Gemfile

```diff
@@ -4,6 +4,10 @@ source 'https://rubygems.org'
 
 gem 'onlyoffice_digitalocean_wrapper'
 gem 'rake'
+gem 'net-ssh', '~> 6.1'
```

Comment:
I think you don't need to specify '~> 6.1'

It's enough that version is fixed in `Gemfile.lock`